CMakeLists.txt

```diff
@@ -39,7 +44,9 @@ else()
     add_compile_options(-Wall)
 endif()
 
-add_subdirectory(src)
+if(WIN32)
```

Comment:
What is the rationale of this if?

Reply:
> What is the rationale of this if?

To build test-dladdr on unix like os to verify the implementation.

Reply:
I see, thanks!